### DISCUSSION TITLE
New discussion space

### DISCUSSION THREAD

**Discussion by btravouillon** [bluebanquise/bluebanquise#432] [General]
Welcome to our new Discussions space!

Should you have any new idea 💡 or issue to configure the stack 😿, this is the place to discuss with the BlueBanquise team 🤓 .

We would love to hear from you. 😁

**Comment by oxedions** [comment]
Yeah !! \o/
At last, discussions tab ! 😊

**Comment by oxedions** [comment]
I added a "cluster deployment" section, for long threads related to users asking help on their cluster deployment, as this kind of exchanges can be on the long run.
